platform/wm/rhodes/browser/CEBrowserEngine.cpp

```diff
@@ -87,6 +87,40 @@ LRESULT CEBrowserEngine::CreateEngine()
 	DWORD			dwFlags;			/// shdocvw client flags
 	IClassFactory	*ppv;				/// pointer variable to hold WebBrowser interface pointer
 
+
+	TCHAR szPlatform[128];
+	memset(szPlatform, 0, 127 * sizeof(TCHAR));
+	SystemParametersInfo(SPI_GETOEMINFO, 127, szPlatform, 0);
+	TCHAR szSIPValue[100];
+	ZeroMemory(szSIPValue,100);
+
+
+		if((_tcsstr(szPlatform,TEXT("MK4000")) != NULL) || (_tcsstr(szPlatform,TEXT("BigBoardMPA3")) != NULL) || (_tcsstr(szPlatform,TEXT("MC18")) != NULL))
```

Comment:
Updated with MK3100 and MK3000

From: Darryn [mailto:notifications@github.com]
Sent: Friday, January 23, 2015 6:10 PM
To: rhomobile/rhodes
Cc: RAGHAVENDRA RAO, LAKSHMI NARASIMMAN
Subject: Re: [rhodes] EMBPD00161219 (#473)


In platform/wm/rhodes/browser/CEBrowserEngine.cpp<https://github.com/rhomobile/rhodes/pull/473#discussion_r23447210>:

> @@ -87,6 +87,40 @@ LRESULT CEBrowserEngine::CreateEngine()

>       DWORD                  dwFlags;                       /// shdocvw client flags

>       IClassFactory  *ppv;                          /// pointer variable to hold WebBrowser interface pointer

>

> +

> +     TCHAR szPlatform[128];

> +     memset(szPlatform, 0, 127 * sizeof(TCHAR));

> +     SystemParametersInfo(SPI_GETOEMINFO, 127, szPlatform, 0);

> +     TCHAR szSIPValue[100];

> +     ZeroMemory(szSIPValue,100);

> +

> +

> +            if((_tcsstr(szPlatform,TEXT("MK4000")) != NULL) || (_tcsstr(szPlatform,TEXT("BigBoardMPA3")) != NULL) || (_tcsstr(szPlatform,TEXT("MC18")) != NULL))

Just use MK3100 and MK3000... not sure what 'BigBoardMPA3' is but I suspect it is a pre-release BSP

—
Reply to this email directly or view it on GitHub<https://github.com/rhomobile/rhodes/pull/473/files#r23447210>.


________________________________
- CONFIDENTIAL-

This email and any files transmitted with it are confidential, and may also be legally privileged. If you are not the intended recipient, you may not review, use, copy, or distribute this message. If you receive this email in error, please notify the sender immediately by reply email and then delete this email.